app/containers/MassEnergizeSuperAdmin/Community/AllCommunities.js

```diff
@@ -58,74 +50,106 @@ class AllCommunities extends React.Component {
     });
   }
 
-
-  getColumns = () => [
-    {
-      name: 'ID',
-      key: 'id',
-      options: {
-        filter: true,
-        filterType: 'textField'
-      }
-    },
+  fashionData(data) {
+    return (
+      data &&
+      data.map((d) => [
+        {
+          id: d.id,
+          image: d.logo,
+          initials: `${d.name && d.name.substring(0, 2).toUpperCase()}`,
+        },
+        smartString(d.name), // limit to first 30 chars
```

Comment:
comment says 30 chars, code would do 60 chars

Reply:
> One thing I'd like to see is when you submit or cancel from an edit action, that you get back to the all actions page with the same filters enabled.

Yes, that was implemented.

Reply:
It doesn't seem to work that way.  If I go to All Actions page, and filter on a community, then edit an action, and cancel, when it comes back to the All Actions page, the actions from all communities are shown.

Reply:
ooooo Wait, I commented on the wrong thing. No, that has not been implemented yet, sorry 🤣 . 
I can do "keeping filters" in a separate PR when I am done with the delete confirmation "live/unlive"